fastlane/lib/fastlane/actions/xcodes.rb

```diff
@@ -0,0 +1,150 @@
+                                       default_value: user,
+                                       default_value_dynamic: true,
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :password,
+                                       short_option: "-p",
+                                       env_names: ["FASTLANE_PASSWORD", "XCODES_PASSWORD"],
+                                       description: "Your Apple ID password",
+                                       code_gen_sensitive: true,
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :update_list,
+                                       env_name: "FL_XCODES_UPDATE_LIST",
+                                       description: "Whether the list of available Xcode versions should be updated before running the install command",
+                                       type: Boolean,
+                                       default_value: true),
+          FastlaneCore::ConfigItem.new(key: :select_for_current_build_only,
+                                       env_name: "FL_XCODES_SELECT_FOR_CURRENT_BUILD_ONLY",
+                                       description: "When true, it won't attempt to install an Xcode version, just find the installed Xcode version that best matches the passed version argument, and select it for the current build steps. It doesn't change the global Xcode version (e.g. via 'xcrun xcode-select'), which would require sudo permissions — when this option is true, this action doesn't require sudo permissions",
+                                       type: Boolean,
+                                       default_value: false),
+          FastlaneCore::ConfigItem.new(key: :binary_path,
+                                       env_name: "FL_XCODES_BINARY_PATH",
+                                       description: "Where the xcodes binary lives on your system (full path)",
+                                       default_value: Helper::XcodesHelper.find_xcodes_binary_path,
+                                       default_value_dynamic: true,
+                                       verify_block: proc do |value|
+                                         UI.user_error!("Couldn't find xcodes binary at path '#{value}'") unless File.exist?(value)
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :xcodes_args,
+                                       env_name: "FL_XCODES_ARGS",
+                                       description: "Pass in xcodes command line arguments directly. When present, other parameters are ignored and only this parameter is used to build the command to be executed",
+                                       type: :shell_string,
+                                       optional: true)
+        ]
+      end
+
+      def self.output
+        [
+          ['XCODES_XCODE_PATH', 'The path to the newly installed Xcode']
```

Comment:
```suggestion
          ['XCODES_XCODE_PATH', 'The path to the newly installed Xcode version']
```

for consistency with the `return_value`.